pkg/util/version/const.go

```diff
@@ -4,3 +4,37 @@ package version
 // Licensed under the Apache License 2.0.
 
 var GitCommit = "unknown"
+
+// InstallStream describes stream we are defaulting to for all new clusters
+var InstallStream = Stream{
+	Version:    NewVersion(4, 4, 10),
+	PullSpec:   "quay.io/openshift-release-dev/ocp-release@sha256:0d1ffca302ae55d32574b38438c148d33c2a8a05c8daf97eeb13e9ab948174f7",
+	MustGather: "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:06ae5b7e36f23eb2e5ae5826499978de4e0124e33938c2e532ed73563b1f7c14",
+}
+
+// Streams describes list of streams we support for upgrades
+var (
+	Streams = []Stream{
```

Comment:
Why does this var have parens? (Genuine question, I am a go noob.)

Reply:
It just happens to and it didn't get cleaned up.  In Go you can say
```
var (
	x = 1
	y = 2
)
```
or
```
var x = 1
var y = 2
```

see https://golang.org/ref/spec#Variable_declarations